barretenberg/cpp/src/barretenberg/smt_verification/smt_polynomials.test.cpp

```diff
@@ -1,76 +1,50 @@
-#include "barretenberg/crypto/generators/generator_data.hpp"
-#include "barretenberg/crypto/pedersen_commitment/pedersen.hpp"
-#include "barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp"
-
 #include <chrono>
 #include <fstream>
 #include <gtest/gtest.h>
 #include <iostream>
 #include <string>
 
+#include "barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp"
+#include "barretenberg/serialize/cbind.hpp"
 #include "barretenberg/stdlib/primitives/field/field.hpp"
 
-#include "barretenberg/serialize/cbind.hpp"
 #include "barretenberg/smt_verification/circuit/circuit.hpp"
+#include "barretenberg/smt_verification/util/smt_util.hpp"
 
 using namespace bb;
 using namespace smt_circuit;
 
-using field_ct = stdlib::field_t<StandardCircuitBuilder>;
+using field_t = stdlib::field_t<StandardCircuitBuilder>;
 using witness_t = stdlib::witness_t<StandardCircuitBuilder>;
 using pub_witness_t = stdlib::public_witness_t<StandardCircuitBuilder>;
 
-// TODO(alex): z1 = z2, s1=s2, but coefficients are not public
 namespace {
 auto& engine = numeric::get_debug_randomness();
 }
 
-msgpack::sbuffer create_circuit(size_t n, bool pub_coeffs)
+msgpack::sbuffer create_polynomial_evaluation_circuit(size_t n, bool pub_coeffs)
```

Comment:
What do we need this for?

Reply:
I'm using this function in the two tests below. Since it is quite large, I decided to take it out separately.